scania-digital-design-system/tegel
# test(text-field): disabled unit test

**Describe pull-request**  
Addin unit test for the disabled state of text-field component

**How to test**  
1. Check the disabled test commit
2. Compare it with [Unit test spreadsheet](https://scaniaazureservices-my.sharepoint.com/:x:/r/personal/tatyana_bumazhkina_scania_com/Documents/Unit%20test%20scenarios.xlsx?d=wca48bf6d92cf4e8c8f456337e2002ec4&csf=1&web=1&e=0Prvks&nav=MTVfezA2MzcyM0JBLUEyNkQtNEQyMy1BNUUwLTlFMzJCQ0E1OEQ4OX0)
3. Run test to see if it passes, try to change something in disabled/index.html to see if it will fail.

**Additional context**  
I had to increase pixel tolerance of table component and do new screenshots as it keeps failing, at least until we merge other PRs that are fixing it.
